### PR TITLE
Add IP Display to HUD, Translation Bugfixes

### PR DESCRIPTION
**Pull request by PassiHD2004**
Added IP Display to Ingame-HUD
Fixed Translation Bug of Zoom button in the settings menu

### CODE REVIEW

File: src/main/java/net/foxes4life/foxclient/Main.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ void initConfig () {
         KonfigCategory misc = new KonfigCategory("misc");
         misc.addEntry("discord-rpc", true);
         misc.addEntry("discord-rpc-show-ip", true);
-        misc.addEntry("smoothzoom", true);
+        misc.addEntry("zoom", true);
```
Comment (RoootTheFox) [inline src/main/java/net/foxes4life/foxclient/Main.java:44]: Why is this changed? Having the config key as just "zoom" might be misleading since its for smooth zoom and not zoom in general

File: src/main/java/net/foxes4life/foxclient/util/ZoomUtils.java [hunk 2]
```diff
@@ -39,7 +39,7 @@ public static void smoothCam () {
     }
 
     public static void calculateZoom(CallbackInfoReturnable<Double> cir) {
-        ZoomUtils.actualZoomLevel = (boolean)Main.konfig.get("misc", "smoothzoom") ?
+        ZoomUtils.actualZoomLevel = (boolean)Main.konfig.get("misc", "zoom") ?
```
Comment (RoootTheFox) [inline src/main/java/net/foxes4life/foxclient/util/ZoomUtils.java:42]: same as before, might be misleading

**Review by RoootTheFox — CHANGES_REQUESTED** [review]
Looks good, has a few changes that should not have been changed though. Good first PR tho

**Comment by Flustix** [comment]
maybe next time keep one pr to only one change

File: src/main/java/net/foxes4life/foxclient/util/ZoomUtils.java (hunk 2 above)
Comment (RoootTheFox) [reply src/main/java/net/foxes4life/foxclient/util/ZoomUtils.java]: looks good now

File: src/main/java/net/foxes4life/foxclient/Main.java (hunk 1 above)
Comment (RoootTheFox) [reply src/main/java/net/foxes4life/foxclient/Main.java]: :+1:

**Review by RoootTheFox — APPROVED** [review]
Everything looks good now
